challenge-03/README.md

```diff
@@ -35,103 +41,122 @@ valor dessa propriedade a quantidade passada por parâmetro;
 - Ele deverá modificar o valor da propriedade `andando` para o valor
 booleano que representa "verdadeiro";
 */
-?
+pessoa.andar = function( x ){
+  if( x !== undefined ){
```

Comment:
-1
Não é necessário fazer verificações por enquanto. Vamos assumir (por enquanto) que, sempre que o método for chamado, um parâmetro será passado a ele :)